<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yemekekle</class>
 <widget class="QMainWindow" name="yemekekle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>591</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <family>A Year Without Rain</family>
             <pointsize>10</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 500 10pt &quot;A Year Without Rain&quot;;
color: black;
font-weight: 800
</string>
           </property>
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="yemek_id">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 2px solid #f4cccc;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>A Year Without Rain</family>
             <pointsize>10</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 500 10pt &quot;A Year Without Rain&quot;;
color: black;
font-weight: 800</string>
           </property>
           <property name="text">
            <string>Yemek Türü</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="yemek_turu">
           <property name="styleSheet">
            <string notr="true">border: 2px solid #f4cccc;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>A Year Without Rain</family>
             <pointsize>10</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 500 10pt &quot;A Year Without Rain&quot;;
color: black;
font-weight: 800</string>
           </property>
           <property name="text">
            <string>Yemek Yöresi</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="yemek_yoresi">
           <property name="styleSheet">
            <string notr="true">border: 2px solid #f4cccc;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>5</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>A Year Without Rain</family>
             <pointsize>10</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 500 10pt &quot;A Year Without Rain&quot;;
color: black;
font-weight: 800</string>
           </property>
           <property name="text">
            <string>Yemek Adı</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="yemek_adi">
           <property name="styleSheet">
            <string notr="true">border: 2px solid #f4cccc;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>A Year Without Rain</family>
           <pointsize>10</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 500 10pt &quot;A Year Without Rain&quot;;
color: black;
font-weight: 800</string>
         </property>
         <property name="text">
          <string>Yemek Tarifi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="yemek_tarifi">
         <property name="styleSheet">
          <string notr="true">border: 2px solid #f4cccc;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_guncelle">
         <property name="styleSheet">
          <string notr="true">QPushButton {
font: 500 10pt &quot;Arimo Medium&quot;;
color: white;
background: #f4cccc;
color:white;
border: 2px solid #f4cccc;
padding: 5px;}
QPushButton:hover {
background-color: #ea9999;
border: 2px solid #ea9999;
}
QPushButton:pressed {
background-color: #e06666;
border: 2px solid #e06666;
}</string>
         </property>
         <property name="text">
          <string>Güncelle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_sil">
         <property name="styleSheet">
          <string notr="true">QPushButton {
font: 500 10pt &quot;Arimo Medium&quot;;
color: white;
background: #f4cccc;
color:white;
border: 2px solid #f4cccc;
padding: 5px;}
QPushButton:hover {
background-color: #ea9999;
border: 2px solid #ea9999;
}
QPushButton:pressed {
background-color: #e06666;
border: 2px solid #e06666;
}</string>
         </property>
         <property name="text">
          <string>Sil</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_getir">
         <property name="styleSheet">
          <string notr="true">QPushButton {
font: 500 10pt &quot;Arimo Medium&quot;;
color: white;
background: #f4cccc;
color:white;
border: 2px solid #f4cccc;
padding: 5px;}
QPushButton:hover {
background-color: #ea9999;
border: 2px solid #ea9999;
}
QPushButton:pressed {
background-color: #e06666;
border: 2px solid #e06666;
}</string>
         </property>
         <property name="text">
          <string>Getir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_yeni">
         <property name="styleSheet">
          <string notr="true">QPushButton {
font: 500 10pt &quot;Arimo Medium&quot;;
color: white;
background: #f4cccc;
color:white;
border: 2px solid #f4cccc;
padding: 5px;}
QPushButton:hover {
background-color: #ea9999;
border: 2px solid #ea9999;
}
QPushButton:pressed {
background-color: #e06666;
border: 2px solid #e06666;
}</string>
         </property>
         <property name="text">
          <string>Yeni</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_kaydet">
         <property name="styleSheet">
          <string notr="true">QPushButton {
font: 500 10pt &quot;Arimo Medium&quot;;
color: white;
background: #f4cccc;
color:white;
border: 2px solid #f4cccc;
padding: 5px;}
QPushButton:hover {
background-color: #ea9999;
border: 2px solid #ea9999;
}
QPushButton:pressed {
background-color: #e06666;
border: 2px solid #e06666;
}</string>
         </property>
         <property name="text">
          <string>Kaydet</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>591</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QTableWidget{background: white;}
QTableWidget::item{background: white;  border: 2px solid #f4cccc;}
QTableWidget::item::pressed{background:  #e06666;}
QTableWidget::item::hover{background:  #e06666;}
QHeaderView::section { background-color:#f4cccc; color:#e06666; font-weight:800}
QTableView QTableCornerButton::section{ background-color:#f4cccc }
QTableWidget::item:selected{ background-color: #e06666 }
</string>
    </property>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>5</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
